README.md

```diff
@@ -24,6 +24,9 @@ daysUntilStale: 60
 # Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
 daysUntilClose: 7
 
+#Only issues or pull requests with all of these labels are check if stale. Set to `[]` to disable
```

Comment:
Suggestion to clarify that this shouldn't have to be explicitly set to `[]` since it's the default:

```suggestion
# Only issues or pull requests with all of these labels are check if stale. Defaults to `[]` (disabled)
```